dace/codegen/compiled_sdfg.py

```diff
@@ -516,13 +510,15 @@ def _construct_args(self, kwargs) -> Tuple[Tuple[Any], Tuple[Any]]:
                     if atype.optional is False:  # If array cannot be None
                         raise TypeError(f'Passing a None value to a non-optional array in argument "{a}"')
                     # Otherwise, None values are passed as null pointers below
+                elif isinstance(arg, ctypes._Pointer):
```

Comment:
Could we leave a note here about the use of an underscored type? I am mostly concerned about the developers changing or removing it in a future version.

Reply:
Interestingly, it is part of the Python stable API: https://docs.python.org/3/library/ctypes.html#ctypes._Pointer